extensions/interactions/FractionInput/directives/oppia-interactive-fraction-input.component.ts

```diff
@@ -185,7 +188,8 @@ export class InteractiveFractionInputComponent implements OnInit, OnDestroy {
           fraction, this.fractionInputRulesService);
       }
     } catch (parsingError) {
-      this.errorMessageI18nKey = parsingError.message;
+      let error = parsingError as Error;
+      this.errorMessageI18nKey = error.message;
```

Comment:
We should unify how we handle errors, as somewhere we do `error isinstance Error` somewhere we use `as`, this is probably something that you should fix in your M2.

Reply:
Done everywhere. Use `instanceof Error` because it make more sense.